OAuth/ResourceOwner/WindowsLiveResourceOwner.php

```diff
@@ -45,4 +47,33 @@ protected function configureOptions(OptionsResolver $resolver)
             'scope' => 'wl.signin',
         ));
     }
+
+    public function getUserInformation(array $accessToken, array $extraParameters = array())
```

Comment:
Missing doc (@inheritdoc should be fine).

Reply:
Thanks for letting me know the required changes. Hopefully this will fix them.